.travis.yml

```diff
@@ -1,16 +1,28 @@
 language: php
 
 php:
-  - "5.5"
-  - "5.6"
-  - "7.0"
-
-before_install:
- - wget http://cs.sensiolabs.org/get/php-cs-fixer.phar
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - nightly
 
 before_script:
- - composer install --dev
+  - travis_retry composer self-update
+  - travis_retry composer install --no-interaction --prefer-source
+
+branches:
+  only:
+    master
 
 script:
- - (cd test ; ../vendor/bin/phpunit -c phpunit.xml)
- - output=$(php php-cs-fixer.phar fix -v --dry-run --level=psr2 .); if [[ $output ]]; then while read -r line; do echo -e "\e[00;31m$line\e[00m"; done <<< "$output"; false; fi;
+  - php -n -d memory_limit=768M ./vendor/bin/php-cs-fixer fix --dry-run -vv --rules=@PSR2 .
```

Comment:
I assume that the non-zero exit status if php-cs-fixer finds something (due to --dry-run) will cause the build to fail?

Reply:
Yeah, it will: https://travis-ci.org/diablomedia/PhlyRestfully/jobs/333360996#L633